src/otflow.jl

```diff
@@ -0,0 +1,98 @@
+struct OTFlow <: AbstractLuxLayer
+    d::Int          # Input dimension
+    m::Int          # Hidden dimension
+    r::Int          # Rank for low-rank approximation
+end
+
+# Constructor with default rank
+OTFlow(d::Int, m::Int; r::Int=min(10,d)) = OTFlow(d, m, r)
+
+# Initialize parameters and states
+function Lux.initialparameters(rng::AbstractRNG, l::OTFlow)
+    w = randn(rng, Float32, l.m) .* 0.01
```

Comment:
this iwll still construct Float64. You are doing Float43 * Float64, make them 0.01f0